src/utils/__tests__/__snapshots__/createSchema.test.ts.snap

```diff
@@ -19,10 +19,7 @@ type Collection {
   descriptionMarkdown: String
 
   # Collection has its description available as markdown
-  hasMarkdownDescription: Boolean!
```

Comment:
the commenting here is not quite right - we're deleting the correct comment and leaving the wrong one?